RadixWallet/Features/SettingsFeature/Preferences/Preferences.swift

```diff
@@ -104,6 +105,13 @@ public struct Preferences: Sendable, FeatureReducer {
 				try await appPreferencesClient.updatePreferences(preferences)
 			}
 
+		case let .advancedLockToogled(isEnabled):
+			state.appPreferences?.security.isAdvancedLockEnabled = isEnabled
```

Comment:
hmm, not sure what how it is done in other apps - should ask users to authenticate at least when they are disabling the Advanced Lock?

Reply:
Indeed, it will be an improvement. I believe it's sufficient to request biometrics only when users are disabling Advanced Lock, as there's no practical reason to require biometrics when enabling it. 
Here's a video showing the result:

https://github.com/user-attachments/assets/0d626bb3-6f20-4102-a82c-4c7baf937fb2